src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    public abstract double area();
+    public abstract String draw();
+
+    private Double sideA;
```

Comment:
Do you have side's in all figures? What about a circle?

Reply:
No  - extract sideA from Figure class